### PR TITLE
OPN-333 Add a new federated_date_modified field to the open data schema

### PR DESCRIPTION
**Pull request by thriuin**

### CODE REVIEW

File: ckanext/canada/schemas/dataset.yaml [hunk 1]
```diff
@@ -226,6 +226,7 @@ dataset_fields:
 - preset: canada_frequency
 - preset: canada_date_published
 - preset: canada_date_modified
+- preset: canada_federated_date_modified
```
Comment (wardi) [inline ckanext/canada/schemas/dataset.yaml:229]: better to add this like and with the other "FGP fields" below. Recommend also adding date validation like the default date preset e.g.
```yaml
- field_name: federated_date_modified
  label: ...
  form_snippet: null
  validators: scheming_required isodate convert_to_json_if_date
```

Comment (thriuin) [reply ckanext/canada/schemas/dataset.yaml:229]: Perfect, thanks.

**Comment by thriuin** [comment]
@wardi could you review again? Thanks

**Comment by wardi** [comment]
@thriuin looks fine to me

**Comment by thriuin** [comment]
Thanks @wardi

**Pull request merged**